ApexOSA/powermeter.py

```
@@ -0,0 +1,85 @@
+                self.__Connexion.close()
+                raise ApexError(APXXXX_ERROR_VARIABLE_NOT_DEFINED, "self.__Unit")
+        else:
+            Command = "SPMEASDETECTORDBM1\n"
+            Send(self.__Connexion, Command)
+            Power = Receive(self.__Connexion)[:-1]
+
+            for p in Power.split("_"):
+                try:
+                    Power = float(p)
+                except:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)